GraphConsoleAppV3/Constants.cs

```diff
@@ -2,12 +2,26 @@
 {
     internal class Constants
     {
-        public const string TenantName = "GraphDir1.onMicrosoft.com";
-        public const string TenantId = "4fd2b2f2-ea27-4fe5-a8f3-7b1a7c975f34";
-        public const string ClientId = "118473c2-7619-46e3-a8e4-6da8d5f56e12";
-        public const string ClientSecret = "hOrJ0r0TZ4GQ3obp+vk3FZ7JBVP+TX353kNo6QwNq7Q=";
-        public const string ClientIdForUserAuthn = "66133929-66a4-4edc-aaee-13b04b03207d";
-        public const string AuthString = "https://login.microsoftonline.com/" + TenantName;
+        public const string TenantNameAppOnly = "calvliTest.onMicrosoft.com";
```

Comment:
>calvliTest.onMicrosoft.com [](start = 49, length = 26)

are you planning to keep your personal tenant details here? Or can we make the sample work with GraphDir1.onMicrosoft.com tenant?